src/SearchField/index.scss

```diff
@@ -11,17 +11,18 @@
     background-color: $search-form-background-color;
   }
 
-  .btn:focus-visible {
+  .pgn__searchfield__action-btn:focus {
     outline: none;
     position: relative;
-    transition: $input-transition;
 
-    &::after {
+    &::before {
       content: "";
-      border: $search-border-focus-width double $search-border-focus-color;
+      border: $search-border-focus-width solid $search-border-focus-color;
       position: absolute;
-      width: 100%;
-      height: 100%;
+      top: -5px;
+      right: -5px;
+      bottom: -5px;
+      left: -5px;
```

Comment:
Please, try to use `button-focus` mixin
https://github.com/openedx/paragon/blob/master/src/Button/index.scss#L134